[2024-11-11T07:15:25.702+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-11T07:15:25.738+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_financial_reports.fetch_financial_reports_task manual__2024-11-11T07:15:23.298130+00:00 [queued]>
[2024-11-11T07:15:25.751+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_financial_reports.fetch_financial_reports_task manual__2024-11-11T07:15:23.298130+00:00 [queued]>
[2024-11-11T07:15:25.752+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-11T07:15:25.778+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_financial_reports_task> on 2024-11-11 07:15:23.298130+00:00
[2024-11-11T07:15:25.792+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=577) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-11T07:15:25.793+0000] {standard_task_runner.py:72} INFO - Started process 589 to run task
[2024-11-11T07:15:25.793+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_financial_reports', 'fetch_financial_reports_task', 'manual__2024-11-11T07:15:23.298130+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/web_scraper.py', '--cfg-path', '/tmp/tmp9guzsv_d']
[2024-11-11T07:15:25.797+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask fetch_financial_reports_task
[2024-11-11T07:15:25.894+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_financial_reports.fetch_financial_reports_task manual__2024-11-11T07:15:23.298130+00:00 [running]> on host d6783a5c6023
[2024-11-11T07:15:26.047+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_financial_reports' AIRFLOW_CTX_TASK_ID='fetch_financial_reports_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-11T07:15:23.298130+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-11T07:15:23.298130+00:00'
[2024-11-11T07:15:26.050+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-11T07:15:26.078+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/web_scraper.py", line 66, in scrape_and_upload_reports
    fetch_cdbg_dr_reports(REPORTS_URL, target_file_name, bucket_name)
TypeError: fetch_cdbg_dr_reports() missing 1 required positional argument: 'folder_name'
[2024-11-11T07:15:26.097+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=fetch_financial_reports, task_id=fetch_financial_reports_task, run_id=manual__2024-11-11T07:15:23.298130+00:00, execution_date=20241111T071523, start_date=20241111T071525, end_date=20241111T071526
[2024-11-11T07:15:26.114+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-11T07:15:26.115+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 10 for task fetch_financial_reports_task (fetch_cdbg_dr_reports() missing 1 required positional argument: 'folder_name'; 589)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/web_scraper.py", line 66, in scrape_and_upload_reports
    fetch_cdbg_dr_reports(REPORTS_URL, target_file_name, bucket_name)
TypeError: fetch_cdbg_dr_reports() missing 1 required positional argument: 'folder_name'
[2024-11-11T07:15:26.132+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-11T07:15:26.158+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-11T07:15:26.161+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
